src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,39 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private final int index0 = 0;
+    private final int index1 = 1;
+    private final int index2 = 2;
+    private final int index3 = 3;
```

Comment:
Please, follow the rules for naming Constants.
Example:
private static final int WORKING_HOURS_INDEX = 2;